capstone/src/main/webapp/data_viz.js

```diff
@@ -21,6 +21,22 @@ const STATES_DISPLAYED = ['Alabama','Alaska','Arizona','Arkansas','California','
   'Oklahoma','Oregon','Pennsylvania','Rhode Island','South Carolina','South Dakota','Tennessee',
   'Texas','Utah','Vermont','Virginia','Washington','West Virginia','Wisconsin','Wyoming'];
 
+//Populations data is from https://www.infoplease.com/us/states/state-population-by-rank
```

Comment:
If you've found a linter to run, might be good to run it on this to fix the spacing (ex: add space between // and comment text), but not a big issue.